kalite/contentload/__init__.py

```diff
@@ -0,0 +1 @@
+# DO NOT PUT ANYTHING HERE
```

Comment:
Because it's an __init__.py file so anything you put in there will be
loaded when importing sub-modules and that's really annoying and has caused
lots of head aches already.. so I would really recommend not to do this
until we're done cleaning up the codebase :) Which we will some day :)


On 6 May 2015 at 18:34, Aron Fyodor Asor <notifications@github.com> wrote:

> In kalite/contentload/__init__.py
> <https://github.com/learningequality/ka-lite/pull/3579#discussion_r29776252>
> :
>
> > @@ -0,0 +1 @@
> > +# DO NOT PUT ANYTHING HERE
>
> Why are we not putting anything here? There's a dearth of explanations on
> this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3579/files#r29776252>.
>